src/pwa/pages/maps.tsx

```diff
@@ -1,6 +1,40 @@
+import { Grid, Typography } from '@mui/material';
+import GoogleMapReact from 'google-map-react';
+import { useEffect, useState } from "react";
 const Map = () => {
+    const [currentGeoPosition, setCurrentGeoPosition] = useState<any>();
+
+    const getCurrentPosition = () => {
+        navigator.geolocation.getCurrentPosition((position) => {
+            console.info(position)
+            setCurrentGeoPosition(position)
+        });
+    }
+
+    useEffect(() => {
+        getCurrentPosition();
+    }, []);
+
     return (
-        <h4>Map</h4>
+        <div style={{ height: '100vh', width: '100%' }}>
+            {currentGeoPosition ?
+                <GoogleMapReact
```

Comment:
Would it be possible to mark current location on the map?

Reply:
Yes, I add a masker as current location on the map